test/Benchmarks/Trace/ActivityCreationBenchmarks.cs

```diff
@@ -8,20 +8,19 @@
 using OpenTelemetry.Trace;
 
 /*
-BenchmarkDotNet v0.13.10, Windows 11 (10.0.23424.1000)
-Intel Core i7-9700 CPU 3.00GHz, 1 CPU, 8 logical and 8 physical cores
+BenchmarkDotNet v0.13.10, Windows 11 (10.0.22621.2861)
+11th Gen Intel Core i7-1185G7 3.00GHz, 1 CPU, 8 logical and 4 physical cores
 .NET SDK 8.0.100
   [Host]     : .NET 8.0.0 (8.0.23.53103), X64 RyuJIT AVX2
   DefaultJob : .NET 8.0.0 (8.0.23.53103), X64 RyuJIT AVX2
 
 
-| Method                                         | Mean      | Error    | StdDev   | Gen0   | Allocated |
-|----------------------------------------------- |----------:|---------:|---------:|-------:|----------:|
-| CreateActivity_NoopProcessor                   | 307.12 ns | 5.769 ns | 6.172 ns | 0.0663 |     416 B |
-| CreateActivity_WithParentContext_NoopProcessor |  75.18 ns | 0.399 ns | 0.354 ns |      - |         - |
-| CreateActivity_WithParentId_NoopProcessor      | 156.52 ns | 1.609 ns | 1.426 ns | 0.0229 |     144 B |
-| CreateActivity_WithAttributes_NoopProcessor    | 372.34 ns | 6.215 ns | 4.852 ns | 0.0992 |     624 B |
-| CreateActivity_WithKind_NoopProcessor          | 302.24 ns | 5.859 ns | 8.402 ns | 0.0663 |     416 B |
+| Method                                         | Mean      | Error    | StdDev    | Median    |
```

Comment:
Include the memory allocation column?

Reply:
coming in next set, as i need a more stable machine !